evc-sumo/Dockerfile

```diff
@@ -0,0 +1,41 @@
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+FROM ubuntu:20.04
+
+# Set environment variables for non-interactive installation
+ARG EVC_TOKEN="NULL"
```

Comment:
If you add `ENV evc_token=${EVC_TOKEN}` that will define an environment variable "evc_token" with the value from your EVC_TOKEN arg.

Reply:
Added ENV to docker file